conformance_tests/tools/metrics/src/test_metric.cpp

```diff
@@ -1217,8 +1217,10 @@ TEST_F(
     zetMetricStreamerTest,
     GivenValidMetricGroupWhenTimerBasedStreamerIsCreatedThenExpectStreamerToSucceed) {
 
-  // The time in seconds for the buffer to overflow would be 2 * (notifyEveryNReports * (samplingPeriod/nanoSecToSeconds)) for this test it will be 512 seconds
-  uint32_t notifyEveryNReports = 256;
+  /* The time in seconds for the buffer to overflow would be 2 * (notifyEveryNReports * (samplingPeriod/nanoSecToSeconds)) for this test it will be 18000 seconds
```

Comment:
Minor: is this math coming from the fact that our implementation uses the mid buffer interrupt ?  if so, does not fit very accurately in the CTS since should be platform and implementation agnostic.

Reply:
removed implementation specifics